src/api_access.ts

```diff
@@ -12,10 +12,71 @@ async function fetchAllPages<T>(endpoint: string, params: any = {}): Promise<T[]
   return octokit.paginate(endpoint, params);
 }
 
+
+
+// Function to fetch the number of commits in a repository
+async function fetchCommitCount(owner: string, repo: string): Promise<number> {
```

Comment:
Merge in main. These are now methods within RepoStats.

Reply:
do you want me to delete them ?